beets/autotag/match.py

```diff
@@ -370,7 +370,7 @@ def _add_candidate(items, results, info):
 
 
 def tag_album(items, search_artist=None, search_album=None,
-              search_id=None):
+              search_ids=[]):
```

Comment:
We should probably describe `search_ids` in the docstring. (Although I note we didn't document `search_id` before—bad @sampsyo!)